
export function handleEpicError(error, caught) {

  return caught;
}
